python_modules/dagster-graphql/dagster_graphql/implementation/execution/__init__.py

```diff
@@ -151,6 +151,11 @@ def _get_error_observable(observer):
 
     def _handle_events(payload):
         events, loading_past = payload
+        events = [
+            event
+            for event in events
+            if event.dagster_event_type != DagsterEventType.ASSET_MATERIALIZATION_PLANNED
```

Comment:
[1]

Reply:
I forgot about the offset cursors...  I guess easiest thing to do is to just filter these events client-side?

Reply:
ya, that or fix pagination to not be offset based 🙂